docs/reference/r-listeners.md

```diff
@@ -0,0 +1,25 @@
+# Kafka Listeners
+
+Charmed Kafka comes with a set of listeners that can be enabled to allow for
+inter- and intra-cluster communication. 
+
+*Internal listeners* are used for internal traffic and exchange of information 
+between Kafka brokers, whereas *external listeners* are used for external clients
```

Comment:
I'm a bit confused about the way line breaks are handled overall in these .md files - I'm assuming that they won't get rendered as newlines on discourse that would interrupt the text.

Reply:
Uhm, good point. I was under the assumption that this would be fine. In normal markdown it should not matter and I did a quick test, and rendering in Discourse seems robust against single line breaks.

My IDE is automatically splitting lines at a certain length. I suppose I could disable this if there is a strong prefernce to have things on a single line. I don't mind too much

Reply:
Me neither :) If discourse renders it fine then I don't see an issue, since this can usually be configured in peoples' editors to their preference.